### ISSUE TITLE
Requirement file rework not reflected for docker

### ISSUE THREAD

**Issue by MNarath1** [SillyTavern/SillyTavern-Extras#231]
Currently the docker compose and Dockerfile don't seem to be adapted for the change how requirement files are handled as an example the full install thats commented out is still with requirement-complete.txt even trough that file doesn't exist anymore and it doesn't at all even mention rvc nor its not dedicated requirement file.
Also pretty sure anything regarding rvc and other stuff are not part of any volume so won't survive a container rebuild

**Comment by MNarath1** [comment]
> It's possible to specify multiple requirement files like this: https://github.com/SillyTavern/SillyTavern-Extras/blob/main/docker/docker-compose.yml#L21
> 
> Can't say anything about volumes, I don't have enough Docker expertise for that.

Yeah i figured that out thankfully i am currently working to get all the modules to work inside docker and add them as templates to the docker composer file(including the changes needed so that you can actually connect to the docker while in bridge mode) and am pretty much done RVC works now fine in it and i added the volumes only problem now is that for some reason it doesn't add the proper folders into the volumes for the models folder under data/models so i had to manually add those folders for the docker to work properly but after i get that done i will probably make a PR.
So basically my current docker setup survives trough rebuilds of the docker but it still doesn't properly sets itself up on first time container build.

**Comment by MNarath1** [comment]
Without the proper folders the docker will just runtime since rvc does not seem to have a fallback if the folder inside data/models doesn't exist

**Comment by mrguymiah** [comment]
It's worth noting that, even with the updated requirements-rvc.txt, I'm getting an INFO error requesting I install tensorboardX. Not sure if that's required to get the functionality up and going. Working on this all myself. I'd appreciate any info you can give on this.

![image](https://github.com/SillyTavern/SillyTavern-Extras/assets/1274076/d24b2f6d-908f-48a2-9a41-26480778ab1d)
